src/pages/kyunghyun/Login/Login.js

```diff
@@ -1,8 +1,174 @@
 import React from 'react';
-
+// import { Link } from 'react-router-dom';
+import './Login.scss';
+import '../../../styles/common.scss';
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      id: '',
+      pw: '',
+      alert: '',
+      valid: false,
+    };
+  }
+
+  handleInput = e => {
+    const { value, name } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  }; // id/pw 통합 input핸들러 : state객체 속성을 이용 []:value 방식으로 적용되는 input을 나눔
+
+  checkValid = e => {
+    const { id, pw } = this.state;
+    const validId = id.includes('@');
+    const validPw = pw.length >= 6;
+    validId && validPw
+      ? this.setState({ valid: true })
+      : this.setState({ valid: false });
+  };
+  // id input의 @와 pw input이 길이가 6 이상의 문자열을 받는 유효성 확인
+  // valid가 true일 시 className이 변경되어 버튼 색 변경
+
+  loginValidation = () => {
+    const { valid, pw, id, alert } = this.state;
+    const { history } = this.props;
+
+    valid ? history.push('/Main-KyungHyun') : (id = '');
+    pw = '';
+    alert =
+      '입력한 사용자 이름을 사용하는 계정을 찾을 수 없습니다. 사용자 이름을 확인하고 다시 시도하세요.';
+  };
+  //// loginValidation은 유효성 검사 함수 -> 추후 스케줄에 적용 예정
+
+  // enterValidation = e => {
+  //   const { history } = this.props;
+  //   if (e.key === 'Enter' && this.state.valid) {
+  //     history.push('/Main-KyungHyun');
+  //   }
+  // };
+
+  handleSubmit = e => {
+    // const { history } = this.props;
+    const { id, pw, alert, valid } = this.state;
+    const { history } = this.props;
+    // e.preventDefault();
+    if (valid) {
+      // 10.58.4.21:8000/users/signup
+      // http://10.58.5.12:8000/users/login
+      fetch('http://10.58.5.12:8000/users/login', {
+        method: 'POST',
+        body: JSON.stringify({
+          email: this.state.id,
+          password: this.state.pw,
+          // name: 'dfsfdsdf',
+          // phone_number: '01041560647',
+          // information: 'EKFKEKEKEK',
+        }),
+      })
+        .then(res => res.json())
+        .then(res => {
+          if (res.token) {
+            history.push('/Main-KyungHyun');
+            localStorage.setItem('kich-token', res.token);
+          } else {
+            this.setState({
+              id: '',
+              pw: '',
+              alert:
+                '입력한 사용자 이름을 사용하는 계정을 찾을 수 없습니다. 사용자 이름을 확인하고 다시 시도하세요.',
+            });
+          }
+        });
+    }
+  };
+  //submit이 일어날 시  history.push('/Main-KyungHyun'); 실행
+
   render() {
-    return null;
+    const { id, pw, valid } = this.state;
+    return (
```

Comment:
```suggestion
    const { id, pw, valid } = this.state;
    
    return (
```
이런 자리는 한줄 띄워주시면 보기 좋을거 같습니다